script.js

```diff
@@ -0,0 +1,40 @@
+const Web3 = require("web3");
```

Comment:
is this code you used for testing? It doesn't look to me like it contains any useful logic, so it can be deleted.